ModuleDescriptor.json

```diff
@@ -16,19 +16,19 @@
         },
         {
           "methods": [ "GET" ],
-          "pathPattern": "/eholdings/jsonapi/vendors*"
+          "pathPattern": "/eholdings/vendors*"
         },
         {
           "methods": [ "GET", "PUT" ],
-          "pathPattern": "/eholdings/jsonapi/packages*"
+          "pathPattern": "/eholdings/packages*"
         },
         {
           "methods": [ "GET" ],
-          "pathPattern": "/eholdings/jsonapi/titles*"
+          "pathPattern": "/eholdings/titles*"
         },
         {
           "methods": [ "GET", "PUT" ],
-          "pathPattern": "/eholdings/jsonapi/customer-resources*"
+          "pathPattern": "/eholdings/customer-resources*"
         },
         {
           "methods": [ "GET", "POST", "PUT", "PATCH", "DELETE" ],
```

Comment:
the routes below these should probably be adjusted for the new proxy endpoints, if we keep them at all

Reply:
Routes in module descriptor updated to include `proxy`.